index.js

```diff
@@ -10,6 +10,11 @@ const isPlainObj = require('is-plain-obj');
 
 const readFile = promisify(fs.readFile);
 
+const detectEOF = file => {
+	const detected = file.match(/\r?\n$/);
+	return detected ? detected[0] : '';
```

Comment:
Doesn't really make sense to use the result, as we won't preserve `\r\n` for the rest of the JSON anyway, so you can just use `\n`.

Reply:
Good point